src/App.tsx

```diff
@@ -2,7 +2,7 @@ import './assets/scss/variables.scss';
 import { HashRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
 import { DefaultLayout } from './layouts/default.layout';
 import { HomePage } from './pages/home-page/HomePage';
-import { PageNotFound } from './pages/pagenotfound/PageNotFound';
```

Comment:
This is actually the correct for now. Please keep the old path './pages/pagenotfound/PageNotFound'